apps/gadgetron/connection/Loader.h

```diff
@@ -38,6 +38,7 @@ namespace Gadgetron::Server::Connection {
 
         template<class FACTORY>
         FACTORY& load_factory(const std::string &prefix, const std::string &classname, const std::string &dll) {
+            GINFO_STREAM("loading " << prefix << " - " << classname << " from the dll " << dll);
```

Comment:
Don't think you need this?

Reply:
No harm to print out these information ? It gives hints if loading classes failed and does not occupy much lines in the log.